applications/zpc/components/zwave_command_classes/src/zwave_command_class_notification.cpp

```diff
@@ -354,6 +354,23 @@ static sl_status_t
     = attribute_store_get_first_child_by_type(
       ep_node,
       ATTRIBUTE(SUPPORTED_NOTIFICATION_TYPES));
+  // Delete notification_types_node when there are no supported notification types.
```

Comment:
just curious, wouldnt it be possible to avoid creation of node instead of remove it ?

Reply:
Node is created away ahead in the inclusion process. I am not sure if it is feasible to avoid creation.

Reply:
The issue was more that the variable length bitmask was not processed correctly, and therefore the supported type was not populated correctly. 
It was fixed in 1.5.0.

Note that this solution does not feel right, you are "removing the device functionality" by erasing the supported types... and there is still a risk that this `SUPPORTED_NOTIFICATION_TYPES` attribute get re-spawned at each system restart by `zwave_command_class_notification_on_version_attribute_update()`

I'd suggest to close this PR.